bundles/org.openhab.binding.radiobrowser/README.md

```diff
@@ -0,0 +1,92 @@
+
+## Channels
+
+| Channel   | Type   | Read/Write | Description                                                                                                 |
+|-----------|--------|------------|-------------------------------------------------------------------------------------------------------------|
+| country   | String | RW         | This allows you to only find stations in ALL or a country of your choice.                                   |
+| state     | String | RW         | When a country is selected, this will auto populate with states that are in your country.                   |
+| language  | String | RW         | You can limit the stations to only be in your language, or you can also use Metadata to set your own list.  |
+| genre     | String | RW         | A list of common genres to help you find a station you like. State Options Metadata allows you to change this. |
+| station   | String | RW         | These are the search results back from the database that match your filter settings.                        |
+| stream    | String | RW         | This is the URL for the selected station.                                                                   |
+| name      | String | RW         | This is the name of the selected station.                                                                   |
+| icon      | String | RW         | This is the icon for the selected station if available in their database.                                   |
+
+## Station Searches
+
+Searches can be done in a few different ways and since the binding will auto select the first result, you can change what is sent to the top of the list by changing the filters config from including `order=clickcount` (default) to `votes`, `clicktrend` or even `random`.
+
+Examples on how to do searches from rules, or you can also change an item to take input by using `oh-input-item` using metadata called `Default list item widget`.
+
+
+Search for all stations that contain `hit` in their name, and auto select the first result.
+
+```
+Radio_Station.sendCommand("hit")
+```
+
+Search and auto select the station if you know the UUID from the website.
+
+```
+Radio_Station.sendCommand("b6a490e8-f498-4a7c-b024-607b3d997614")
+```
+
+Clear any manual search results using the above two methods, and `REFRESH` back to using the normal filter channels.
+
+```
+Radio_Station.sendCommand(REFRESH)
+```
+
+## Show Your Thanks
+
+If you enjoy the binding, then please consider sponsoring or giving a once off tip as a thank you via the links.
+This allows me to purchase software and hardware to contribute more bindings, and also shows this binding should gain more feature like favourites.
+Some coffee to keep me coding faster never hurts :slight_smile:
+
+Sponsor @Skinah on GitHub
+<https://github.com/sponsors/Skinah/>
+
+Paypal can also be used via
+matt A-T pcmus D-O-T C-O-M
```

Comment:
While i wish you a lot of donations and coffee, I'm not sure if this is allowed to be added to the official documentation of the openhab website. Need some advice from other @openhab/add-ons-maintainers if this is oke or not.

Reply:
Have removed this from the readme and will instead reply on the widget to make this suggestion.